pconnector/client/client.go

```diff
@@ -34,11 +35,12 @@ import (
 func New(
 	logger hclog.Logger,
 	path string,
+	connectorUtilsAddr string,
```

Comment:
Why don't we just create an `Option` that lets you set a single env variable? Then we can set any number of variables in Conduit.

Reply:
The `Option`s are applied to the plugin client config, and the target (and token which I still need to add here) aren't really about that.

Reply:
... on the other hand, I think I can make the options more generic so that we can configure both "ends".

Reply:
Nevermind, I just realized `Cmd` is available as a field in `plugin.ClientConfig`, so it's easy to add an option for this. :+1: